README.md

```diff
@@ -1 +1,3 @@
-# advent-of-code-2020
\ No newline at end of file
+# 🎅 My Advent of Code 2020 Solutions 🧑‍🎄
```

Comment:
```suggestion
# 🎅 My Advent of Code 2020 Kek Solutions 🧑‍🎄
```

Reply:
Hi @Alexdruso , great edit! Love it, keep it up!
If you are interested I have a start-up that sells kek factors solutions. We are looking for a Cobol developer that would help us adding keks into banks. Would you be interested?

Reply:
I have always been into Finkek! Sure, I want to be working on this new exciting kektional language!